cmd/noe/main.go

```diff
@@ -22,21 +24,43 @@ import (
 )
 
 func main() {
-	var preferredArch, systemOS string
+	var preferredArch, schedulableArchs, systemOS string
 	var metricsAddr string
 	var registryProxies, matchNodeLabels string
 
 	flag.StringVar(&preferredArch, "preferred-arch", "amd64", "Preferred architecture when placing pods")
+	flag.StringVar(&schedulableArchs, "cluster-schedulable-archs", "amd64", "Comma separated list of architectures schedulable in the cluster")
```

Comment:
in the description we say is empty but here we set a default of amd64

Reply:
Which description?

Reply:
PR description

```
Approach
Allow passing a comma-separated list of architectures schedulable in
the cluster. As we detect image's architecture support, we ignore
those architectures that have not been specified as supported by the
cluster.

This new flag is optional, so we do not change existing behaviour if
it is not specified (empty string).```

Reply:
I'll clarify. I meant that if you specify it as an empty string we do not change current behaviour.

Reply:
Added clarification to the description.

Side note: I'm not sure if we should default to `amd64` to match the default preferred architecture or if it should default to being deactivated (empty string).